From f6514f6cb92260a7729b819f5719f90d281b3932 Mon Sep 17 00:00:00 2001
Message-Id: <f6514f6cb92260a7729b819f5719f90d281b3932.1690410491.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1690410490.git.sharlatanus@gmail.com>
References: <cover.1690410490.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 20 Jun 2023 21:57:26 +0100
Subject: [PATCH v2 17/22] gnu: python-reproject: Update to 0.11.0.

* gnu/packages/astronomy.scm (python-reproject): Update to 0.11.0.
[native-inputs]: Move python-pyvo, python-gwcs, python-shapely from here ...
[propagated-inputs]: ... to here. Add python-click, python-cloudpickle,
python-dask, python-fsspec, python-zarr.
---
 gnu/packages/astronomy.scm | 25 +++++++++++++++----------
 1 file changed, 15 insertions(+), 10 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 6f1910232b..1735eaa25e 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2058,13 +2058,13 @@ (define-public python-regions
 (define-public python-reproject
   (package
     (name "python-reproject")
-    (version "0.10.0")
+    (version "0.11.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "reproject" version))
        (sha256
-        (base32 "1ha0a1ja7k09ysd05adffgsapfwzc6m6az34a0av2mhmlwy4zb1q"))))
+        (base32 "0p07qlqinb826m4n5b5invyfyv4z750sai2caqaf598mgj04l61p"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -2087,21 +2087,26 @@ (define-public python-reproject
             (lambda _
               (setenv "HOME" (getcwd)))))))
     (propagated-inputs
-     (list python-astropy
+     (list python-asdf
+           python-astropy
            python-astropy-healpix
+           python-click
+           python-cloudpickle
+           python-dask
+           python-fsspec
+           python-gwcs
            python-numpy
-           python-scipy))
+           python-pyvo
+           python-scipy
+           python-shapely
+           python-zarr))
     (native-inputs
-     (list python-asdf
-           python-cython
+     (list python-cython
            python-extension-helpers
-           python-gwcs
            python-pytest-astropy
-           python-pyvo
            python-semantic-version
            python-pytest
-           python-setuptools-scm
-           python-shapely))
+           python-setuptools-scm))
     (home-page "https://reproject.readthedocs.io")
     (synopsis "Astronomical image reprojection in Python")
     (description
-- 
2.40.1

